packages/fx-core/src/plugins/resource/frontend/index.ts

```diff
@@ -49,7 +50,7 @@ export class FrontendPlugin implements Plugin, ArmResourcePlugin {
 
   public async scaffold(ctx: PluginContext): Promise<TeamsFxResult> {
     if (isVsCallingCli()) {
-      return ok(undefined);
+      throw new NotImplemented();
```

Comment:
Since the bicep file is pack in zip file in VS extension, what if there is any breaking/schema change in Bicep file after we release VS extension in December?

Reply:
Then we need to update the zip, other wise VS toolkit breaks. I think bicep should consider backward compatibility for any change after go production. @blackchoey